templates/workspace_services/azureml/terraform/acr.tf

```diff
@@ -1,12 +1,10 @@
-
-
 resource "azurerm_container_registry" "acr" {
   name                          = local.acr_name
   location                      = data.azurerm_resource_group.ws.location
   resource_group_name           = data.azurerm_resource_group.ws.name
   sku                           = "Premium"
   admin_enabled                 = false
-  public_network_access_enabled = false
+  public_network_access_enabled = var.public_access_enabled
```

Comment:
as discussed, I suspect this might benefit from a comment

Reply:
agreed - have added comment to explain